cmd/backup-manager/app/util/util.go

```diff
@@ -395,6 +400,313 @@ func GetBRMetaData(ctx context.Context, provider v1alpha1.StorageProvider) (*kvb
 	return backupMeta, nil
 }
 
+// CalcBackupSizeFromBackupmeta get backup size from backup meta
+func CalcBackupSizeFromBackupmeta(ctx context.Context, provider v1alpha1.StorageProvider) (int64, error) {
+	// read all snapshots from backup meta file
+	volSnapshots, err := getSnapshotsFromBackupmeta(ctx, provider)
+	if err != nil {
+		return 0, err
+	}
+	// get all snapshots per backup volume from aws
+	snapshots, err := getBackupVolSnapshots(volSnapshots)
+
+	if err != nil {
+		return 0, err
+	}
+
+	backupSize, err := calcBackupVolSnapshotSize(volSnapshots, snapshots)
+
+	if err != nil {
+		return 0, err
+	}
+
+	return int64(backupSize), nil
+}
+
+// getSnapshotsFromBackupmeta read all snapshots from backupmeta
+// return volume - snapshot map
+func getSnapshotsFromBackupmeta(ctx context.Context, provider v1alpha1.StorageProvider) (map[string]string, error) {
+	newVolumeIDMap := make(map[string]string)
```

Comment:
Why add 'new' in the `newVolumeIDMap` variable?

Reply:
fixed in https://github.com/pingcap/tidb-operator/pull/4819/commits/fc18e21a421dee08cf3257099b503047e5d552a7